### PR TITLE
Add Google Translate component

### PR DESCRIPTION
**Pull request by chrisrhymes**
- Add a Google Translate component, [inspiration from here](https://codesandbox.io/p/sandbox/google-translate-in-react-js-qzdjj?file=%2Fsrc%2FApp.js)
- Add option to display the Google Translate component in the [footer](https://deploy-preview-485--northants-design-system.netlify.app/?path=/story/library-structure-footer--example-footer-with-translate)

## ToDo
- [x] Get a list of languages for the dropdown and update dropdown.

## Testing
- Checkout this branch and run `npm run dev`
- View the Components -> Google Translate examples and test out the Google translate dropdown to see the same text translated
- View the Structure -> Footer examples and see the Example Footer With Google Translate

### CODE REVIEW

File: src/library/components/GoogleTranslate/GoogleTranslate.types.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export const GoogleTranslateLanguages = [
+  { label: 'Bengali', value: 'bn' },
+  { label: 'English', value: 'en' },
+  { label: 'Gujarati', value: 'gu' },
+  { label: 'Lithuanian', value: 'lt' },
+  { label: 'Polish', value: 'pl' },
+  { label: 'Portuguese', value: 'pt' },
+  { label: 'Punjabi', value: 'pa' },
+  { label: 'Romanian', value: 'ro' },
+  { label: 'Russian', value: 'ru' },
```
Comment (DamienBurkeWN) [inline src/library/components/GoogleTranslate/GoogleTranslate.types.ts:28]: Politically given Russian cyber attacks on this very site I'd say sod them for being included.
Put Ukrainian in instead!

Comment (DamienBurkeWN) [reply src/library/components/GoogleTranslate/GoogleTranslate.types.ts:28]: Actually there's a lot of obvious ones missing - is this selection based upon any research of local community?

File: src/library/pages/NewsArticle/NewsArticle.tsx [hunk 2]
```diff
@@ -31,55 +32,95 @@ export const NewsArticle: React.FC<NewsArticleProps> = ({ withImage, imagePlaceh
 
         {withImage && <NewsArticleImage image1440x810={withImage} image144x81={imagePlaceholder} />}
 
-        <p><strong>Pupils who are eligible for Free School Meals will receive support from Northamptonshire County Council during the February Half Term.</strong></p>
-        <p>As stated in December, prior to the Christmas break, the council will use some of the funding it has received from the government’s COVID Winter Grant Scheme to offer help.</p>
+        <p>
+          <strong>
+            Pupils who are eligible for Free School Meals will receive support from Northamptonshire County Council
```
Comment (DamienBurkeWN) [inline src/library/pages/NewsArticle/NewsArticle.tsx:37]: Should be West Northamptonshire Council really ;)

File: src/library/pages/ServiceLandingPageExample/ServiceLandingPageExample.storydata.tsx [hunk 3]
```diff
@@ -204,9 +204,9 @@ export const promoHeroImageData = {
   headline: 'Sustainable West Northants',
   content: '<p>We have collated a variety of small changes that can have a big impact for you to try today.</p>',
   imageLarge:
-    'https://cms.westnorthants.gov.uk/sites/default/files/styles/responsive/public/1440/810/0/2021-12/Abington_Park_1.jpg',
+    'https://cms.westnorthants.gov.uk/sites/default/files/styles/responsive/public/1600/500/0/2022-02/Daventry%20Country%20Park.JPG',
   imageSmall:
-    'https://cms.westnorthants.gov.uk/sites/default/files/styles/responsive/public/144/81/0/2021-12/Abington_Park_1.jpg',
+    'https://cms.westnorthants.gov.uk/sites/default/files/styles/responsive/public/1600/500/0/2022-02/Daventry%20Country%20Park.JPG',
```
Comment (DamienBurkeWN) [inline src/library/pages/ServiceLandingPageExample/ServiceLandingPageExample.storydata.tsx:209]: I know it's just an example page but should the small image be the same as the large?

File: src/library/structure/HeroImage/HeroImage.storydata.ts [hunk 4]
```diff
@@ -8,9 +8,9 @@ export const HeroImageExampleBoxedData: HeroImageProps = {
   callToActionText: 'Find out more',
   callToActionURL: 'https://www.northamptonshireparks.co.uk/brixworth-country-park/Pages/default.aspx',
   imageLarge:
-    'https://cms.westnorthants.gov.uk/sites/default/files/styles/responsive/public/1440/810/0/2021-12/Abington_Park_1.jpg',
+    'https://cms.westnorthants.gov.uk/sites/default/files/styles/responsive/public/1600/500/0/2022-02/Daventry%20Country%20Park.JPG',
   imageSmall:
-    'https://cms.westnorthants.gov.uk/sites/default/files/styles/responsive/public/144/81/0/2021-12/Abington_Park_1.jpg',
+    'https://cms.westnorthants.gov.uk/sites/default/files/styles/responsive/public/1600/500/0/2022-02/Daventry%20Country%20Park.JPG',
```
Comment (DamienBurkeWN) [inline src/library/structure/HeroImage/HeroImage.storydata.ts:13]: Same question on image sizes throughout this file

File: src/themes/GlobalStyleReset.tsx [hunk 5]
```diff
@@ -154,4 +154,11 @@ export const GlobalStyleReset: any = createGlobalStyle<any>`
   table tr td {
     vertical-align: top;
   }
+  body {
+    position: static;
+    top: 0px !important;
+  }
+  iframe.skiptranslate {
+    display: none !important;
```
Comment (DamienBurkeWN) [inline src/themes/GlobalStyleReset.tsx:162]: 🤨

**Review by DamienBurkeWN — APPROVED** [review]
Works well - comments for clearing up when you do the language list

File: src/themes/GlobalStyleReset.tsx (hunk 5 above)
Comment (chrisrhymes) [reply src/themes/GlobalStyleReset.tsx:162]: This hides the bar at the top of the page that the script inserts.

File: src/library/components/GoogleTranslate/GoogleTranslate.types.ts (hunk 1 above)
Comment (chrisrhymes) [reply src/library/components/GoogleTranslate/GoogleTranslate.types.ts:28]: Yes, supplied from the project team.

**Pull request merged**